.github/workflows/tests.yml

```diff
@@ -115,6 +115,23 @@ jobs:
         run: |
           pytest vetiver/tests/test_sklearn.py
 
+  test-pydantic-old:
```

Comment:
adding tests for pre/post pydantic v2 until all dependencies (`spacy` does not support v2) support v2